mino/minogrpc/certs/mem.go

```diff
@@ -90,15 +90,16 @@ func (s *InMemoryStore) Fetch(addr Dialable, hash []byte) error {
 
 	conn.Close()
 
+	// we can assume that `peers` is not empty
```

Comment:
This is mentioned in the documentation of `PeerCertificates`: "On the client side, it can't be empty".
It doesn't explain why, but I guess that if the handshake is successful then there is at least one certificate that has been exchanged.